up42/tasking.py

```diff
@@ -105,5 +106,54 @@ def construct_order_parameters(
 
         return order_parameters
 
+    def _query_paginated_output(self, url: str):
+        page = 0
+        response = self.auth._request(request_type="GET", url=url)
+        json_results = response["content"]
+        not_last_page = not response["last"]
+        while not_last_page:
+            page += 1
+            url = replace_page_query(url, page)
+            response = self.auth._request(request_type="GET", url=url)
+            json_results.extend(response["content"])
+            not_last_page = not response["last"]
+        return json_results
+
+    def get_quotations(
+        self,
+        workspace_id: Optional[str] = None,
+        order_id: Optional[str] = None,
+        decision: Optional[str] = None,
+        sortby: str = "createdAt",
+        descending: bool = True,
+    ):
+        """
+        This function returns the quotations for tasking by filtering and sorting by different parameters.
+
+        Args:
+            workspace_id (Optional[str], optional): The workspace id (uuid) to filter the search.
+            order_id (Optional[str], optional): The order id (uuid) to filter the search.
+            decision (Optional[str], optional): the status of the quotation (NOT_DECIDED, ACCEPTED or REJECTED).
+            sortby (str, optional): Arranges elements in asc or desc order based on a chosen field.
+            The format is <field name>,<asc or desc>.
+            descending (bool, optional): Descending or ascending sort.
+
+        Returns:
+            JSON: The json representation with the quotations resulted from the search.
+        """
+        sort = f"{sortby},{'desc' if descending else 'asc'}"
+        url = f"{self.auth._endpoint()}/v2/tasking/quotation?page=0&sort={sort}"
+        if workspace_id is not None:
+            url += f"&workspaceId={workspace_id}"
+        if order_id is not None:
+            url += f"&order_id={order_id}"
```

Comment:
CMIIW but if I look from swagger, the param name is `orderId`,  not `order_id`
https://docs.up42.dev/openapi/swagger-ui/?urls.primaryName=order-service#/tasking/listTaskingQuotation

Reply:
DONE! Thanks